src/main/java/org/zalando/jackson/datatype/money/MoneyModule.java

```diff
@@ -11,11 +12,13 @@
 import javax.money.CurrencyUnit;
 import javax.money.MonetaryAmount;
 import java.util.Currency;
+import javax.money.NumberValue;
 
 import static com.fasterxml.jackson.core.util.VersionUtil.mavenVersionFor;
 
 public final class MoneyModule extends Module {
 
+    private final JsonSerializer<NumberValue> numberValueSerializer;
```

Comment:
If one wants to change MonetaryAmount JSON structure, e.g. introduce value+scale fromat it would be easier just to implement custom MonetaryAmountSerializer from scratch.
FieldNames are designed for current structure, extending it to support something different does not worth the effort.